krun/scheduler.py

```diff
@@ -483,6 +483,12 @@ def _make_post_cmd_env(self, results):
     def run(self):
         """Benchmark execution starts here"""
 
+        # In reboot mode, wait for the system to come up before we proceed
+        if self.platform.hardware_reboots:
```

Comment:
Do we need to check whether there are jobs left in the manifest, or has that already happened by this point?

Reply:
There's an assertion a few lines down which tells me that it is assured that there are jobs left if we get here:

https://github.com/softdevteam/krun/blob/bfa452a688899b488bb4b17314293d504e259382/krun/scheduler.py#L486-L496

Reply:
OK, cool